src/Components/Audio/Audio.js

```diff
@@ -1,13 +1,39 @@
-import React from "react";
+import React, {useState, useEffect} from "react";
 import './Audio.css'
 import ReactAudioPlayer from "react-audio-player";
 
-const Audio = ({audioURL}) => {
+const Audio = ({audioURL, difficulty, turn}) => {
+
+  const[audioClicks, setAudioClicks] = useState(0)
+  const[disable,setDisable] = useState("show")
+
+  useEffect(()=>{
+    setDisable("show")
+    if(difficulty === "easy"){
+      setAudioClicks(3)
+    } else if (difficulty === "medium") {
+      setAudioClicks(2)
+    } else {
+      setAudioClicks(1)
+    }
+  },[turn])
+
+  const handleAudioClick = () => {
+    if (audioClicks > 1){
+      setAudioClicks(audioClicks-1)
+      console.log("bruh")
+    } else {
```

Comment:
Maybe we should remove the `console.log(bruh)`?

Reply:
It has been removed